docs/development.md

```diff
@@ -41,6 +41,26 @@ flowchart
     bors_process --> BorsContext
 ```
 
+## Run tests
+Here's how you can run the tests for the project:
+
+```console
+# Start the database
+docker-compose up -d
+
+# Set the database URL to the test database. Used both by sqlx and the tests.
```

Comment:
We already have documentation about the DB below, can we somehow unify it? Otherwise it will be duplicated and easy to go out of sync. We can just add a section on testing below the existing DB documentation.